### ISSUE TITLE
Crash: No static method addAll(Ljava/lang/Iterable;Ljava/util/List;)V in class AbstractMessageLite

### ISSUE THREAD

**Issue by josher93** [adform/adform-tracking-android-sdk#3]
Crashes the app, this is what I get:

`FATAL EXCEPTION: main
                                                                                        Process: com.globalpaysolutions.yocomprorecarga, PID: 20479
                                                                                        java.lang.NoSuchMethodError: No static method addAll(Ljava/lang/Iterable;Ljava/util/List;)V in class Lcom/google/protobuf/AbstractMessageLite$Builder; or its super classes (declaration of 'com.google.protobuf.AbstractMessageLite$Builder' appears in /data/app/com.globalpaysolutions.yocomprorecarga-2/split_lib_dependencies_apk.apk)
                                                                                            at com.adform.mobile.contracts.tracking.MobileTrackingDataContract$MobileTrackingData$Builder.addAllProducts(MobileTrackingDataContract.java:1401)
                                                                                            at com.adform.adformtrackingsdk.web.ProtobufBuilder.buildAsProtoBuf(ProtobufBuilder.java:98)
                                                                                            at com.adform.adformtrackingsdk.web.NetworkLoader.load(NetworkLoader.java:77)
                                                                                            at com.adform.adformtrackingsdk.services.SendService.startingSend(SendService.java:112)
                                                                                            at com.adform.adformtrackingsdk.services.SendService$2.run(SendService.java:149)
                                                                                            at android.os.Handler.handleCallback(Handler.java:739)
                                                                                            at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                                            at android.os.Looper.loop(Looper.java:145)
                                                                                            at android.app.ActivityThread.main(ActivityThread.java:6934)
                                                                                            at java.lang.reflect.Method.invoke(Native Method)
                                                                                            at java.lang.reflect.Method.invoke(Method.java:372)
                                                                                            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404)
                                                                                            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199)`

I tried with a newer version of Protobuf (3.5.1) but couldn't make it run

**Comment by wman1980** [comment]
I got the same crash. to be more precise I get that error if I do this:
```
TrackPoint point = new TrackPoint(Constants.AD_FORM_TRACKING_ID);
point.setSectionName("TestSectionName");
AdformTrackingSdk.sendTrackPoint(point);
```

not crashing with adform sdk 1.1.6

**Comment by rprunskas** [comment]
Hi,

fixed in 1.1.8
